srv/salt/_modules/cephprocesses.py

```diff
@@ -29,7 +29,7 @@
              'mgr': ['ceph-mgr'],
              'storage': ['ceph-osd'],
              'mds': ['ceph-mds'],
-             'igw': ['rbd-target-gw', 'rbd-target-api'],
```

Comment:
those processes don't exist in SES5, and during an upgrade the cephprocesses.wait will fail because those processes don't exist.
We can add those processes in the next version of DeepSea.

Reply:
well, but how can you guarantee in which version people will upgrade. some people will do it in one month and some in a year..